app/assets/stylesheets/ama_layout/layout_components/callouts.scss

```diff
@@ -23,22 +24,32 @@
     color: $white;
   }
 
+  &__container{
+    @include grid-row;
+    @include clearfix;
+  }
+
   &__sub-head{
     color: $white;
   }
 
   &__image-container{
-    @include large-2;
+    @include large-1;
+    @include small-4;
   }
 
   &__image{
     display: block;
-    margin-left: auto;
-    margin-right: auto;
+    margin: 0 auto;
+  }
+
+  &__content-container{
+    @include large-11;
+    @include small-8;
   }
 
   &__content{
-    @include large-10;
+    position: relative;
```

Comment:
is there a reason we dropped the large-10? do we need the position relative for something?